src/dockerclient.h

```diff
@@ -0,0 +1,20 @@
+#ifndef AKTUALIZR_LITE_DOCKERCLIENT_H
+#define AKTUALIZR_LITE_DOCKERCLIENT_H
+
+#include <json/json.h>
+#include <string>
+
+namespace Docker {
+
+class DockerClient {
```

Comment:
Not critical, since `DockerClient` is already in the scope of `Docker` namespace then `Docker` prefix becomes redundant in the class name.